fakeredis.py

```diff
@@ -367,18 +366,19 @@ def __init__(self, db=0, charset='utf-8', errors='strict',
         self._encoding_errors = errors
         self._pubsubs = []
         self._decode_responses = decode_responses
-        self._connected = connected
+        self.connected = connected
         if decode_responses:
-            _patch_responses(self)
+            _patch_responses(self, _make_decode_func)
+
+        if not connected:
```

Comment:
Should this be done unconditionally? Otherwise you get the odd behaviour that you can toggle `connected` on and off, but only if it was initially set to false.

Reply:
Yes I believe you are correct. I'll implement that change.